documentation/3.4/content/userguide/istio/istio.md

```diff
@@ -43,9 +47,18 @@ If your applications have suitable tracing code in them, then you will also be a
 use distributed tracing, such as Jaeger, to trace requests across domains and to
 other components and services that have tracing enabled.
 
+WebLogic Kubernetes Operator assume you are familiar with Istio or Red Hat OpenShift Service Mesh. If you are new to
+them, we strongly recommend reading the respective documentations and work through the `Bookinfo`
+sample application to familiar with the mesh and verify it is working properly in your environment before proceeding working with the Operator.
```

Comment:
to familiar -> to familiarize yourself
verify it -> verify that it
environment  -> environment, (comma)
proceeding working with the Operator -> proceeding to work with the operator